samples/react/src/css/styleguide/00-variables.css

```diff
@@ -0,0 +1,9 @@
+:root {
```

Comment:
[question] Why do you have a separate variables files in the styleguide folder? Wouldn't you just use your main variables file?

Reply:
Oops -- I meant to namespace these with `--sg-` in my variable file and in using them in the color swatches. That said, I think there's value in having styleguide-specific vars. For example, if I'm using a light gray color for my color swatch border, I wouldn't want that color to change based on the project-equivalent color variable; I'd want it consistent across projects. Updating to include the namespacing.

Reply:
gotcha, ok